argcomplete/__init__.py

```diff
@@ -222,6 +227,12 @@ def __call__(self, argument_parser, always_complete_options=True, exit_method=os
 
         completions = self._get_completions(comp_words, cword_prefix, cword_prequote, last_wordbreak_pos)
 
+        if dfs:
+            display_completions = {key_part: value
+                                   for key, value in self.get_display_completions().items()
+                                   for key_part in key.split(" ")}
```

Comment:
I think it's possible to have options with spaces, so there would be no help string for them.
Not sure if it is reasonable to fix because it would take much more changes - I have to change `get_display_completions()` method or add very similar method (that would not join options in [_get_option_completions](https://github.com/kislyuk/argcomplete/blob/master/argcomplete/__init__.py#L348) )

Reply:
Issue has been fixed and a test case added. Thanks!